src/main/resources/static/index.js

```diff
@@ -2,63 +2,63 @@
 function initMap() {
 
     // The location of UC
-    const uc = { lat: 39.130146, lng: -84.518982 };

```

Comment:
According to javascript best practices, parentheses or curly braces should not have whitespace in the beginning or towards the end.